tests/test_utils/generate_cluster_desc.hpp

```diff
@@ -7,7 +7,39 @@
 #pragma once
 
 #include <filesystem>
+#include <string>
+#include <iostream>
 
-namespace fs = std::filesystem;
+namespace test_utils {
 
-fs::path GetClusterDescYAML();
+inline std::string GetAbsPath(std::string path_){
+    std::filesystem::path umd_root = std::filesystem::absolute("umd" / std::filesystem::path(__FILE__).parent_path().parent_path().parent_path()).lexically_normal();
```

Comment:
During the `make` build flow, this seems to result in a path e.g. `/home/joel/git/tt-umd/umd/device/bin/silicon/x86/create-ethernet-map` which is invalid.

Reply:
Will deprecate the make flow for unit tests if that's ok